int16 Pos1
int16 Pos2
int16 Pos3
int16 Pos4
int16 Pos5
int16 Pos6
int16 Pos7
int16 Pos8
int16 Pos9
int16 Pos10
int16 Pos11
int16 Pos12
int16 Pos13
int16 Pos14
int16 Pos15
int16 Pos16
int16 Pos17
int16 Pos18
int16 Speed1
int16 Speed2
int16 Speed3
int16 Speed4
int16 Speed5
int16 Speed6
int16 Speed7
int16 Speed8
int16 Speed9
int16 Speed10
int16 Speed11
int16 Speed12
int16 Speed13
int16 Speed14
int16 Speed15
int16 Speed16
int16 Speed17
int16 Speed18
int8 Voltage1
int8 Voltage2
int8 Voltage3
int8 Voltage4
int8 Voltage5
int8 Voltage6
int8 Voltage7
int8 Voltage8
int8 Voltage9
int8 Voltage10
int8 Voltage11
int8 Voltage12
int8 Voltage13
int8 Voltage14
int8 Voltage15
int8 Voltage16
int8 Voltage17
int8 Voltage18
int8 Temperature1
int8 Temperature2
int8 Temperature3
int8 Temperature4
int8 Temperature5
int8 Temperature6
int8 Temperature7
int8 Temperature8
int8 Temperature9
int8 Temperature10
int8 Temperature11
int8 Temperature12
int8 Temperature13
int8 Temperature14
int8 Temperature15
int8 Temperature16
int8 Temperature17
int8 Temperature18
